electronics/README.asciidoc

```diff
@@ -4,10 +4,10 @@
 
 ### Power
 The mainboard is split into 2 isolated sides with seperate ground planes.
-The microcontroller side is powered with 5V by an USB connector that is stepped down to 3.3V by a 5V to 3V voltage regulator (NCP164CSN330T1G). 
-The motor driver side is powered by a 4S LIPO battery (up to 16.8V) and is stepped down to 5V to power the drivers, isolators and servos 
+The microcontroller side is powered with 5V by an USB connector that is stepped down to 3.3V by a 5V to 3.3V voltage regulator (NCP164CSN330T1G). 
+The motor driver side is powered by a 4S LIPO battery (up to 16.8V) and is stepped down to 5V to power the isolators and servos 
 using a buck converter (TPS62933DRLR). Both the buck converter and voltage regulator have power LEDs.
-The board can power 3 brushed DC motors, its encoders, 2 servos and one external IR sensor and IR LED.
+The board can power 3 brushed DC motors with up to 2.95A each, its encoders, 2 servos and one external IR sensor and IR LED.

```

Comment:
How did you calculate 2.95A? I think it's slightly different.

Reply:
Using the datasheet formula:
Vitrip = 2.64V (as set by the 100k resistor on itrip)
Ripropi = 2.7k ohm
Aipropi = 3070 ( i used the wrong constant here)
ITRIP regulation level = VITRIP / RIPROPI X AIPROPI
The problem is now fixed